README.md

```diff
@@ -1,17 +1,16 @@
-# Appwrite SDK for Go
+# Appwrite Go SDK
 
-![License](https://img.shields.io/github/license/appwrite/sdk-for-go.svg?v=1)
-![Version](https://img.shields.io/badge/api%20version-0.5.3-blue.svg?v=1)
+![License](https://img.shields.io/github/license/appwrite/sdk-for-go.svg?style=flat-square)
+![Version](https://img.shields.io/badge/api%20version-1.6.0-blue.svg?style=flat-square)
+[![Build Status](https://img.shields.io/travis/com/appwrite/sdk-generator?style=flat-square)](https://travis-ci.com/appwrite/sdk-generator)
+[![Twitter Account](https://img.shields.io/twitter/follow/appwrite?color=00acee&label=twitter&style=flat-square)](https://twitter.com/appwrite)
+[![Discord](https://img.shields.io/discord/564160730845151244?label=discord&style=flat-square)](https://appwrite.io/discord)
 
-**WORK IN PROGRESS!**
+**This SDK is compatible with Appwrite server version 1.6.x. For older versions, please check [previous releases](https://github.com/appwrite/sdk-for-go/releases).**
```

Comment:
minor thing - SDK could be backwards compatible but there are no older published releases for `1.5.x`, right?